src/test/datascience/interpreters/environmentActivationService.vscode.test.ts

```diff
@@ -86,9 +86,9 @@ suite('DataScience - VSCode Notebook - (Conda Execution) (slow)', function () {
         disposeAllDisposables(disposables);
         traceInfo(`Ended Test (completed) ${this.currentTest?.title}`);
     });
-    suiteTeardown(() => {
+    suiteTeardown(async () => {
         if (originalActiveInterpreter && pythonApi) {
-            pythonApi.environments
+            await pythonApi.environments
```

Comment:
Fix, we need to ensure the active interpreter is reset else other tests could fail if its not setup in time, and could result in flaky tests